Hello, traveler. This is the beginning of a log file called: "Fascist Tweet Archiver".
Some basic system info about the host machine: posix.uname_result(sysname='Linux', nodename='linawesome', release='4.13.0-37-generic', version='#42~16.04.1-Ubuntu SMP Wed Mar 7 16:03:28 UTC 2018', machine='x86_64')
This log was begun Monday, 02 April 2018 at 13:54



INFO: read target account data; we're tracking 59 accounts
WARNING: no ProtocolError (got exception "name 'ProtocolError' is not defined"); trying requests.packages.urllib3.exceptions instead
NOTE: successfully imported from requests
